templates/glue-job-role.yaml

```diff
@@ -12,12 +12,46 @@ Parameters:
     Type: String
     Description: ARN of the SQS queue which we send invalid records to.
 
+  S3IntermediateBucketName:
+    Type: String
+    Description: Name of the S3 intermediate (JSON) bucket
+
+  S3ParquetBucketName:
+    Type: String
+    Description: Name of the S3 Parquet bucket
+
+  S3ArtifactBucketName:
+    Type: String
+    Description: Name of the S3 bucket where cloudformation templates and tests are stored.
+
+
 Resources:
 
   JobRole:
     Type: AWS::IAM::Role
     Properties:
       Policies:
+      - PolicyName: Glue
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - glue:*
+            - cloudwatch:PutMetricData
+            Resource:
+            - "*"
+      - PolicyName: IAM
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - iam:ListRolePolicies
+            - iam:GetRole
+            - iam:GetRolePolicy
+            Resource:
+            - "*"
```

Comment:
Not a huge fan of the * but this was probably what the managed policies were doing.